modules/auxiliary/admin/http/atlassian_confluence_auth_bypass.rb

```diff
@@ -0,0 +1,131 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Atlassian Confluence Data Center and Server Authentication Bypass via Broken Access Control',
+        'Description' => %q{
+          This module exploits an Broken Access Control vulnerability in Atlassian Confluence servers leads to Authentication Bypass.
+          A specially crafted request can be create new admin account without authorization in the Atlassian server.
+        },
+        'Author' => [
+          'Unknown', # exploited in the wild
+          'Emir Polat' # metasploit module
+        ],
+        'References' => [
+          ['CVE', '2023-22515'],
```

Comment:
Could we possibly add a link to the AttackerKB analysis here:
```
https://attackerkb.com/topics/Q5f0ItSzw5/cve-2023-22515/rapid7-analysis
```

Reply:
> Could we possibly add a link to the AttackerKB analysis here:
> 
> ```
> https://attackerkb.com/topics/Q5f0ItSzw5/cve-2023-22515/rapid7-analysis
> ```

I've added the AttackerKB analysis. Commit id: 9219a3e90abbc54ff4da71513e4ab9c2e371d8d4